ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -33,8 +35,46 @@
     }),
 
     computed: {
+      ...mapGetters('interactives', [
+        'currentInteractiveDataLoading',
+        'currentInteractive',
+        'currentInteractiveSession'
+      ]),
+
+      interactiveConfig () {
+        switch (this.currentInteractive.interactiveType) {
+        case 'draggable-statement-completion':
+          return this.currentInteractive.draggableStatementCompletionData
+        case 'insert-code':
+          return this.currentInteractive.insertCodeData
+        case 'draggable-ordering':
+          return this.currentInteractive.draggableOrderingData
+        default:
+          // TODO handle_error_ozaria
+          noty({ text: 'Interactive type is not set for the interactive', type: 'error', timeout: '2000' })
+          throw new Error('Invalid interactive type provided for interactive')
+        }
+      },
+
+      localizedInteractiveConfig () {
+        const interactiveConfigI18n = this.interactiveConfig.i18n || {}
+        const userLocale = me.get('preferredLanguage', true) // TODO drive from store
```

Comment:
TODO left here?

Reply:
TODO is valid we should be using the `me` Vuex store module in Vue components.

Reply:
Updated this as part of the localization ozariaUtils changes.